src/sqlitedb.cpp

```diff
@@ -225,13 +228,59 @@ bool DBBrowserDB::tryEncryptionSettings(const QString& filename, bool* encrypted
         {
             sqlite3_finalize(vm);
 #ifdef ENABLE_SQLCIPHER
+            if (!isDatabasePasswordChecked)
+            {
+                isDatabasePasswordChecked = true;
+
+                QMap<QString, QVariant> databasePasswords = PreferencesDialog::getSettingsValue("db", "databasepasswords").toMap();
```

Comment:
@MKleusberg How do I "convert" from `QMultiMap` to `QVariant`, needed on the line `setSettingsValue("db", "databasepasswords", QVariant(databasePasswords));`?